packages/from-hardhat/tsconfig.json

```diff
@@ -0,0 +1,39 @@
+{
+  "compilerOptions": {
```

Comment:
Do we want things like `strict` and `noImplicitAny` in here? Excluding now it is going to make it rather complicated to standardize on it in the future if we move to TypeScript project builds w/ a single inherited base config.

Reply:
I had to turn `strictNullChecks` off because the types in compile-common aren't suitable for use with that flag

(@eggplantzzz is looking into fixing that)

Reply:
I don't care about `noImplicitAny`. Happy to add it if you think it's useful.

Reply:
I tend to think that `noImplicitAny` is useful, as otherwise it's hard to tell when a type is correctly inferred, vs implicitly `any`.

Reply:
Nice, I love turning on a thing and having no added type-checks. Will push shortly :)